### ISSUE TITLE
[Suggestion & maybe bug]Show price history when using proxy

### ISSUE THREAD

**Issue by KurumiSerori** [jshackles/Enhanced_Steam#1372]
I have an China-steam-wallet account and visit the steam webpage using a proxy(US) which would makes steamCountry in the cookie to be 'US', but actually I set the currency in ES to be 'CNY' and I would like to looking up isthereanydeal.com with region 'CN'. However, the mechanism now will search the price history as USD and than transfer it to CNY, which makes it much bigger than it is in China region.
As you can see below,

View with US IP:
![view with us ip](https://user-images.githubusercontent.com/10769279/27507471-fafa1a08-590a-11e7-92aa-c0d9db6332e5.png)

View with CN IP:
![view with cn ip](https://user-images.githubusercontent.com/10769279/27507472-fda53738-590a-11e7-83c1-58c9554e67a0.png)

**Comment by KurumiSerori** [comment]
I think the problem is that when we are looking up for the price history and comparing it to that on store page, it is supposed that the price we want is under the same steam wallet region.
The method ES is using now seems to check the visiting region(which may not equal to wallet region, especially when not login) and then grab data.
So is there any way we can get the wallet region automatically? I didn't find anything in cookie that can tell me about this. Or maybe just add a way to adjust ES settings to solve this problem.

**Comment by KurumiSerori** [comment]
#1314 refer this issue

**Comment by Luckz** [comment]
This isn't exclusive to using a proxy - it also applies if your Steam wallet currency is different than your current IP country currency.
I mean, you can legitimately be inside a different country :)

**Comment by KurumiSerori** [comment]
That's right. And that's what I think too. The main problem is that the store page currency referred by IP/Cookie is not always the same with wallet currency. And find a way to get wallet currency could be a way to solve this problem. Or a temp scheme here, just fix the API parameter before calling them when the setting of ES includes some special currency like 'CNY'.

**Comment by yume-chan** [comment]
Seems relate to #775

I also have this problem and I found a workaround from how #864 works:

1. Open a steam store page
2. Hit <kbd>F12</kbd>
3. Switch to the `Console` tab
4. Input `document.cookie="fakeCC=cn;path=/;expires=Sat, 31 Dec 2050 00:00:00 GMT"` (only input codes you understand or trust, replace `cn` to the country code you want if needed)
5. Hit <kbd>Enter</kbd>
6. Refresh the page

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
